yellowbrick/features/projection.py

```diff
@@ -172,26 +184,36 @@ def ax(self):
     def ax(self, ax):
         self._ax = ax
 
-    def layout(self):
+    def layout(self, divider=None):
```

Comment:
So the idea is that subclasses can pass up their own divider? E.g. PCA creates its own divider for the heatmap and sends it up to this one to make the colorbar? 

Would you mind adding the parameter to the docstring?

Reply:
Yup. Adding new dividers destroys the plot.
I will add the docstring.